man/nvim.1

```diff
@@ -0,0 +1,417 @@
+If
+.Ar nvimrc
+is
+.Cm NONE ,
+loading plugins is also skipped.
+See
+.Ic :help initialization .
+.It Fl i Ar nviminfo
+Use
+.Ar nviminfo
+instead of the default of
+.Pa ~/.nviminfo .
+If
+.Ar nviminfo
+is
+.Cm NONE ,
+do not read or write an nviminfo file.
+See
+.Ic :help viminfo .
+.It Fl -noplugin
+Skip loading plugins.
+Implied by
+.Cm -u NONE .
+.It Fl o Ns Op Ar N
+Open
+.Ar N
+windows stacked horizontally.
+If
+.Ar N
+is omitted, open one window for each file.
+If
+.Ar N
+is less than the number of file arguments, allocate windows for the first
+.Ar N
+files and hide the rest.
+.It Fl O Ns Op Ar N
+Open
+.Ar N
+windows stacked vertically.
+If
+.Ar N
+is omitted, open one window for each file.
+If
+.Ar N
+is less than the number of file arguments, allocate windows for the first
+.Ar N
+files and hide the rest.
+.It Fl p Ns Op Ar N
+Open
+.Ar N
+tab pages.
+If
+.Ar N
+is omitted, open one tab page for each file.
+If
+.Ar N
+is less than the number of file arguments, allocate tab pages for the first
+.Ar N
+files and hide the rest.
+.It Cm + Ns Op Ar linenum
+For the first file, position the cursor on line
+.Ar linenum .
+If
+.Ar linenum
+is omitted, position the cursor on the last line of the file.
+Note that
+.Cm +5
+and
+.Cm -c 5
+on the command-line are equivalent to
+.Ic :5
+inside
+.Nm .
+.It Cm +/ Ns Op Ar pattern
+For the first file, position the cursor on the first occurrence of
+.Ar pattern .
+If
+.Ar pattern
+is omitted, the most recently used search pattern is used (if there is one).
+Note that
+.Cm /foo
+and
+.Cm -c /foo
+on the command-line are equivalent to
+.Ic /foo
+and
+.Ic :/foo
+inside
+.Nm .
+See
+.Ic :help search-pattern .
+.It Fl -cmd Ar command
+Execute
+.Ar command
+before processing any nvimrc.
+Up to 10 instances of these can be used, independently from instances of
+.Fl c .
+.It Fl c Ar command
+Execute
+.Ar command
+after reading the first file.
+Up to 10 instances of
+.Fl c
+or
+.Cm +
+can be used.
+.It Fl S Op Ar session
+Source
+.Ar session
+after the first file argument has been read.
+Equivalent to
+.Cm -c \(dqsource session\(dq .
+.Ar session
+cannot start with a hyphen.
+If
+.Ar session
+is omitted, then
+.Pa Session.vim ,
+if found, is used.
+See
+.Ic :help session-file .
+.It Fl s Ar scriptin
+Read normal mode commands from
+.Ar scriptin .
+The same can be done with the command
+.Ic :source! scriptin .
+If the end of the file is reached before
+.Nm
+exits, further characters are read from the keyboard.
+.It Fl w Ar scriptout
+Append all typed characters to
+.Ar scriptout .
+Can be used for creating a script to be used with
+.Fl s
+or
+.Ic :source! .
+.It Fl W Ar scriptout
+Like
+.Fl w ,
+but truncate
+.Ar scriptout .
+.It Fl -startuptime Ar file
+During startup, append timing messages to
+.Ar file .
+Can be used to diagnose slow startup times.
+.It Fl -api-info
+Dump API metadata serialized to msgpack and exit.
+.It Fl -embed
+Use standard input and standard output as a msgpack-rpc channel.
```

Comment:
I had it that way before, but changed it since they aren't being talked about in programming contexts (i.e., the actual file descriptors).